test/test_video_feed.py

```diff
@@ -0,0 +1,30 @@
+""" Tests the behavior of the VideoFeed class. """
+
+import pytest
+
+from modules.vision.module.video_feed import VideoFeedCV2
+
+
+class TestVideoFeedCV2:
+    """ Groups the tests for the VideoFeedCV2 class,
+    please connect at least one camera. """
+
+    def test_resolution(self):
+        """ Tests if we are able to get and set the resolution correctly. """
+        w, h = 640, 480
+        vf = VideoFeedCV2(0)
+        vf.set_resolution(w, h)
+        res = vf.get_resolution()
+        assert(res[0] == w)
+        assert(res[1] == h)
+
+    def test_open(self):
```

Comment:
I think you should test whether the device is opened before you test your resolution

Reply:
fixed in https://github.com/R2D2-2019/vision/pull/4/commits/bc5c6c49bab12a2013078fc8b5abd4564266563f